wire-tests/src/test/proto-kotlin/com/squareup/wire/protos/kotlin/MessageUsingMultipleEnums.kt

```diff
@@ -69,6 +69,11 @@ data class MessageUsingMultipleEnums(
           unknownFields = unknownFields
         )
       }
+
+      override fun redact(value: MessageUsingMultipleEnums): MessageUsingMultipleEnums? =
+          value.copy(
+        unknownFields = ByteString.EMPTY
+      )
```

Comment:
I don't like this indent, is it expected behavior or could this be a poet bug?

Reply:
The indentation for arguments is applied manually. It's a missing KP feature that I'm working on.